google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -330,7 +336,8 @@ public InputStream getContent() throws IOException {
           String contentEncoding = this.contentEncoding;
           if (!returnRawInputStream
               && contentEncoding != null
-              && contentEncoding.contains("gzip")) {
+              && (CONTENT_ENCODING_GZIP.equalsIgnoreCase(contentEncoding.trim())
```

Comment:
equalsIgnoreCase needs a character set; in this context UTF-8 probably works.

Reply:
I'm not sure where your concerns lie since Java Strings are UTF-16. Please advise.